packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -125,6 +125,11 @@ module.exports = class SingleSpaReactGenerator extends Generator {
       this.destinationPath(`src/root.component.${srcFileExtension}`),
       templateOptions
     );
+    this.fs.copyTpl(
```

Comment:
I'm not sure if this plays well with Typescript. I'm testing this now.

Reply:
This works just fine with Typescript, but this testing uncovered another issue. The project would not start with TypeScript support because of a plugin constructor invocation problem. That fix is also included in this PR.